group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorMetricsShard.java

```diff
@@ -140,10 +156,23 @@ public GroupCoordinatorMetricsShard(
         this.topicPartition = Objects.requireNonNull(topicPartition);
     }
 
-    public void incrementNumClassicGroups(ClassicGroupState state) {
+    /**
+     * @return The interval to update classicGroupGauges.
+     */
+    public long classicGroupGaugesUpdateIntervalMs() {
```

Comment:
nit: This does not seem necessary. We could just make the constant public or move the constant to where the scheduled task is defined.